### ISSUE TITLE
Autority - celé záhlaví

### ISSUE THREAD

**Issue by krizovj** [proarc/proarc#1352]
Bylo by možné, aby se z databáze autorit stahovalo celé záhlaví záznamů u autorů? Ptž když hledám autora, tak chci záznam autora a ne záznam toho díla a bez toho celého záhlaví to nejde rozlišit. Není tam vůbec stažené podpole t, takže všechny ty záznamy vypadají stejně, až na ten identifikátor. Bylo by super, kdyby to celé záhlaví bylo už vlevo v těch výsledcích vyhledávání Takhle to vypadá v ProArcu:
![autority1](https://user-images.githubusercontent.com/21309201/122733756-c2731900-d27d-11eb-9526-491cca306f53.jpg)

A tohle je ten záznam v databázi autorit: https://aleph.nkp.cz/F/?func=direct&doc_number=001111032&local_base=AUT

**Comment by luckajirku** [comment]
a nejde to na vstupu třídit podle toho, co je ve FMT? jestli se nepletu, tak v tomhle případě nás zajímají jen záznamy s hodnotou JA, ne?

**Comment by krizovj** [comment]
No, můžu taky chtít stáhnout do "subject" to dílo, ne? Když to bude nějaká literární studie o tom díle, tak to můžu chtít použít jako klíčové slovo.

**Comment by kerschfilip** [comment]
Na problém u vyhledávání autorů přes autority jsme narazili i při testování nového klienta pro zpracování eČlánků. Když se najde více autorů se stejným jménem není možné rozklíčovat, který je ten správný.

Spíš než návrh na rozšíření to vidím jako chybu, která znesnadňuje práci.

**Comment by ZdenkaSera** [comment]
Protože se neobjevil žádný komentář v souvislosti s přiřazením štítku "k realizaci", odebírám štítek "návrh na rozšíření".

**Comment by ZdenkaSera** [comment]
Funkční ok (vyzkoušela jsem autory, geografické názvy, věcné třídění). Podle mého o dost lepší, ale záhlaví jsou mnohde stručná a úplně nezachrání,  Závěry ale nechám na ostatních.

**Comment by SykoraLukas** [comment]
Do ProArcu se přebírají jen ta pole, co mají své odpovídající hodnoty v MODSu. Pokud dodáte mapování požadovaných poli do MODSu, můžeme zobrazovat více poli. 

Pozn: šablona pro transformaci z Marc21 do Mods je jen jedna pro všechny druhy stažení a pro všechny instalace. Takže jakákoli změna se dotkne všech...

**Comment by kerschfilip** [comment]
@SykoraLukas díky za vysvětlení. To je docela nepříjemná komplikace, zkusíme se nad tím zamyslet.

**Comment by SykoraLukas** [comment]
Ono by šlo udělat, že se to pole vpravo zobrazí, ale poté by se do MODSu stejně nenatáhlo, protože nemá odpovídající mapování a byl by zase problém, proč to tam je vidět, když se to pak stejně nestáhne ....takže se zobrazuje jen to, co se reálně stáhne do pole v MODSu.

**Comment by kerschfilip** [comment]
@SykoraLukas to by asi řešilo jen případy jmenných autorit, ne? `description` se mimo `name` v MODS myslím nemá používat. 

Bavili jsme se o tom obecně v KNAV a přišlo by nám asi nejlepší, aby se vpravo zobrazovalo co nejvíc informací, zpracovatel si mohl dobře vybrat, který záznam chce použít, ale do MODSu se prostě namapuje jen to, co namapovat jde. Tzn. žádné mapování navíc bychom nevymýšleli.

**Comment by SykoraLukas** [comment]
No to právě nejde. Už vpravo se zobrazuje naparsovaný mods. Takže cokoli chcete mít vpravo tak k tomu musí být šablona z marc21 do Modsu, jak jsem psal výše. Takže ty informace je potřeba předat do nějakého pole (klidně jen dočasného a pak se to dá odmazat při vložení do záznamu MODSu).

**Comment by kerschfilip** [comment]
Aha, vycházeli jsme z toho, jak jsi výše psal

> Ono by šlo udělat, že se to pole vpravo zobrazí, ale poté by se do MODSu stejně nenatáhlo

tak jsem to asi blbě pochopil. V tom případě by mohlo být řešení to nějak nacpat dočasně do toho pole `description` a pak zase smazat, aby to nebylo v datech (protože to nebude úplně správně)

**Comment by ZdenkaSera** [comment]
Issue necháváme otevřené k diskusi/doladění.

**Comment by SykoraLukas** [comment]
Upraveno. Pokud je pole 678 bere se dočasně informace, která se při importu do záznamu odstraní.

**Comment by kerschfilip** [comment]
Zobrazování záhlaví bylo rozšířeno. Případné návrhy na úpravu prosím do nového issue
